src/main/java/com/android/volley/toolbox/HttpHeaderParser.java

```diff
@@ -126,13 +133,25 @@
     public static long parseDateAsEpoch(String dateStr) {
         try {
             // Parse date in RFC1123 format if this header contains one
-            return DateUtils.parseDate(dateStr).getTime();
-        } catch (DateParseException e) {
+            return newRfc1123Formatter().parse(dateStr).getTime();
+        } catch (ParseException e) {
             // Date in invalid format, fallback to 0
```

Comment:
I realize this is out of scope for this review but it seems like this exception should be logged.

Reply:
Agreed; added log.